src/features/CareViewSummary/utils/CareViewSummary.js

```diff
@@ -108,3 +111,81 @@ export const getSlidesPerView = (isMobileView, isTabletView) => {
     return Math.floor((window.outerWidth * 0.9) / 142);
   }
 };
+
+export const setCurrentShiftTimes = (
```

Comment:
Can we leverage setCurrentShiftTimes in DrugChartUtils? - https://github.com/Bahmni/openmrs-module-ipd-frontend/blob/main/src/features/DisplayControls/DrugChart/utils/DrugChartUtils.js#L398

The function can be defined with parameters like

```
export const setCurrentShiftTimes = (
  shiftDetails,
  isReadMode = false,
  visitSummary = ""
) => {
...
}
```